src/components/nav/Header.tsx

```diff
@@ -19,6 +21,7 @@ export function Header() {
         <div className="flex flex-col items-end md:flex-row-reverse md:items-start gap-2">
           <WalletControlBar />
           <TransfersStatusBar />
+          <SideBarMenu isOpen={true} />
```

Comment:
I guess you'll want to manage the open state either with some local state in this component, passing the setter down to the WalletControlBar, or with a flag in the zustand store. Whatever you prefer is fine